DataImport.Server.TransformLoad/Features/FileTransport/FileServer.cs

```diff
@@ -49,6 +49,10 @@ public async Task<IEnumerable<string>> GetFileList(Agent ftpsAgent)
                 {
                     client.ValidateCertificate += OnValidateFtpsCertificate;
                     await client.ConnectAsync();
+                    if (String.IsNullOrEmpty(ftpsAgent.FilePattern))
+                        list.AddRange(from file in await client.GetListingAsync(ftpsAgent.Directory)
+                                      where file.Type == FtpFileSystemObjectType.File
+                                      select file.FullName);
```

Comment:
Please consider changing the Linq style, see https://techdocs.ed-fi.org/pages/viewpage.action?pageId=12158193#CCodingStandards-LINQQueries